### ISSUE TITLE
Option to override header row left, centre, right

### ISSUE THREAD

**Issue by wplit** [soflyy/oxygen-bugs-and-features#332]
**Describe the feature you'd like to see included in Oxygen.**
To be able to turn off the block, so we can directly edit the row left, row center, row right, without using custom CSS.

**What are the use cases for this feature?**
It's particularly difficult to center the logo in row center when the left and right have different widths. The .oxy-header-center just fills in the available space, but without being able to directly give the left and right the same widths, it's often not centered properly. it should be easy to center it, even if there are more menu items on the left than the right and taking up different widths.

**Comment by Netzlichter** [comment]
Such a feature would be great to make the header builder more intuitive without custom css...

**Comment by jamescrabb** [comment]
Much needed to allow better control currently it’s a bit frustrating

**Comment by swinggraphics** [comment]
> Much needed to allow better control currently it’s a bit frustrating

"Maddening" is the word I would use.

**Comment by adrien-robert** [comment]
+1

**Comment by hawk0r** [comment]
+1

**Comment by Mylogo** [comment]
I agree that we need this feature because I had this exact use case. While we wait for this feature to hopefully be implemented, here is a workaround:
### Workaround:
These instructions allow your left header to expand to the full width, removing the other two containers.
1. Create the selector `.header-only-left .oxy-header-center` and set `display: none`
2. Create the selector `.header-only-left .oxy-header-right` and set `display: none`
3. Create the selector `.header-only-left .oxy-header-left` and set `width: 100%`
4. Add the class `header-only-left`to the header row.

You may now add any additional styles by either
- Adding them to the `.header-only-left .oxy-header-left` selector
- **OR** (my preferred way, because you don't need to work on selectors): Add a div to the left div, make it `width: 100%` and apply any styling to it.

Note: Move any elements from the center/right containers to the left container. 
I followed my instructions and put the previous and new website next to each other and noticed absolutely no difference visually. Of course, the center/right divs are still there.
